### ISSUE TITLE
ZF3, ProfilingAdapterFactory does not implement __invoke

### ISSUE THREAD

**Issue by xorock** [bjyoungblood/BjyProfiler#63]
When using with latest ZF3 

``` php
return [
    'service_manager' => [
        'factories' => [
            Adapter::class => ProfilingAdapterFactory::class,
        ],
    ],
];
```

ProfilingAdapterFactory throws error: Fatal error: Class BjyProfiler\Db\Adapter\ProfilingAdapterFactory contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Zend\ServiceManager\Factory\FactoryInterface::__invoke)

**Comment by andrey-mokhov** [comment]
#62 fix it

**Comment by andrey-mokhov** [comment]
for ZF3 can use:
```console
composer require andrey-mokhov/bjy-profiler:dev-master
```

**Comment by andrey-mokhov** [comment]
Released  version 1.3.0

[See changes](https://github.com/andrey-mokhov/bjy-profiler/releases/tag/1.3.0)

```console
$ composer require --dev andrey-mokhov/bjy-profiler:^1.3
```
